hw04-generics/src/main/java/homework/CustomerService.java

```diff
@@ -0,0 +1,40 @@
+package homework;
+
+
+import java.util.AbstractMap;
+import java.util.Map;
+import java.util.TreeMap;
+
+public class CustomerService {
+
+    //todo: 3. надо реализовать методы этого класса
+    //важно подобрать подходящую Map-у, посмотрите на редко используемые методы, они тут полезны
+
+    private TreeMap<Customer, String> _map = new TreeMap<>(
```

Comment:
1) Лучше указывать интерфейс:
```suggestion
    private NavigableMap<Customer, String> map = new TreeMap<>(
```

2) Подчеркивания в именах полей в java не приняты